### PR TITLE
Redirect playground to play.nteract.io

### PR DESCRIPTION
**Pull request by yuvipanda**
The nteract.io team is happy to maintain that, so we should support
them with a redirect rather than running our own instance.

I checked with @rgbkrk and he was happy with the redirect!

### CODE REVIEW

File: config/prod.yaml [hunk 1]
```diff
@@ -55,8 +55,7 @@ redirector:
...
         to: mybinder.readthedocs.io
-
-playground:
-  ingress:
-    hosts:
-      - playground.mybinder.org
+    - type: host
+      host:
+        from: playground.mybinder.org
+        to: play.nteract.io
```
Comment (rgbkrk) [inline config/prod.yaml]: 🎉

File: config/staging.yaml [hunk 2]
```diff
@@ -53,8 +53,7 @@ redirector:
...
         to: mybinder.readthedocs.io
-
-playground:
-  ingress:
-    hosts:
-      - playground.staging.mybinder.org
+    - type: host
+      host:
+        from: playground.staging.mybinder.org
+        to: play.nteract.io
```
Comment (rgbkrk) [inline config/staging.yaml]: we could probably host a staging site or allow a query param that uses staging too

xref: https://github.com/nteract/nteract/issues/2125

**Comment by rgbkrk** [comment]
Great!

Sorry about the lack of quick turnaround on some of the features that would have made this easier, we've mostly been focusing on functionality and maintainability of the frontend first.

File: config/staging.yaml (hunk 2 above)
Comment (yuvipanda) [reply config/staging.yaml]: That would be great!

**Comment by choldgraf** [comment]
I'm +1 on this, I generally trust the nteract team to make things awesome and I am all for cross-promotion in the jupyter/OS world :-)

**Comment by choldgraf** [comment]
that said, perhaps we should get some +1s from other binder team members too

**Comment by yuvipanda** [comment]
Thanks everyone :)

**Pull request merged**